pkg/app/api/analysisresultstore/filestore.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
When we referred to a repository, we may want to write the explanation such as "The following codes are based on ~" from my experience.

WDYT?

Reply:
We only refer to our defined package [pkg/filestore](https://github.com/pipe-cd/pipe/blob/master/pkg/filestore/filestore.go) so I don't think we need that explanation.

Reply:
@ono-max I feel the same way as @khanhtc1202 but you're correct. If you borrow the code from another repository even for just a bit, we have to mention that as:

https://github.com/pipe-cd/pipe/blob/410778ccc3f893495ea6768907df99177ecbbf85/pkg/app/api/httpapi/httpapimetrics/delegator.go#L15-L26